common.php

```diff
@@ -0,0 +1,42 @@
+
+<?php
+
+function _log($id, $action, $message)
+{
+    global $log_file;
+
+    $ip = $_SERVER['REMOTE_ADDR'];
+    $time = date("Y/m/d h:i:s");
+    $page = file_get_contents('_pending_contrib/'.$id.'/page');
```

Comment:
chekc & sanitize $id before calling file_get_contents() with this spoofable _SERVER variable